app/matplotlib.ipynb

```diff
@@ -342,7 +394,16 @@
    "execution_count": 15,
    "outputs": [],
    "source": [
-    "# write your code here\n"
+    "import seaborn as sns\n",
+    "\n",
+    "file_path = \"../data/customer_data.csv\"\n",
+    "data = pd.read_csv(file_path)\n",
+    "\n",
+    "print(data.head())\n",
+    "\n",
+    "sns.pairplot(data, diag_kind=\"kde\", markers=\"o\", plot_kws={\"alpha\": 0.5})\n",
+    "\n",
+    "plt.suptitle(\"Scatter Plot Matrix of Customer Demographics and Behavior\", y=1.02)\n"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the scatter plot matrix is displayed in the notebook.